### ISSUE TITLE
Postgres should support relativeTime query

### ISSUE THREAD

**Issue by cbaker6** [parse-community/parse-server#7746]
### New Feature / Enhancement Checklist

- [x] I am not disclosing a [vulnerability](https://github.com/parse-community/parse-server/blob/master/SECURITY.md).
- [x] I am not just asking a [question](https://github.com/parse-community/.github/blob/master/SUPPORT.md).
- [x] I have searched through [existing issues](https://github.com/parse-community/parse-server/issues?q=is%3Aissue).

### Current Limitation

The PostgresAdapter doesn't support the `relativeTime` query constraint. Currently, the mongo adapter supports it.

### Feature / Enhancement Description

Relative time queries allows devs to query dates based on strings, i.e. `'1 day ago'`

### Example Use Case

Using the Swift SDK:

```swift
let constraint = relative("updatedAt" <= "3 days ago")
let query = GameScore.query(constraint)
do {
  let recentObjects = try await query.find()
  print(recentObjects)
} catch {
  // Handle error
}
```

### Alternatives / Workarounds

Specify query constraints on dates directly.

### 3rd Party References

The parse-server already supports this for Mongo:

https://github.com/parse-community/parse-server/blob/4c29d4d23b67e4abaf25803fe71cae47ce1b5957/spec/ParseQuery.spec.js#L4769-L4868

**Comment by parseplatformorg** [comment]
🎉 This change has been released in version [5.0.0-alpha.14](https://github.com/parse-community/parse-server/releases/tag/5.0.0-alpha.14)

**Comment by parseplatformorg** [comment]
🎉 This change has been released in version [5.0.0-beta.10](https://github.com/parse-community/parse-server/releases/tag/5.0.0-beta.10)

**Comment by parseplatformorg** [comment]
🎉 This change has been released in version [5.1.0](https://github.com/parse-community/parse-server/releases/tag/5.1.0)
